packages/admin/cms-admin/src/dam/DamPage.tsx

```diff
@@ -1,8 +1,56 @@
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
 import * as React from "react";
+import { FormattedMessage } from "react-intl";
+import { useRouteMatch } from "react-router";
 
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { ContentScopeInterface, useContentScope } from "../contentScope/Provider";
+import { useContentScopeConfig } from "../contentScope/useContentScopeConfig";
+import { DamScopeProvider } from "./config/DamScopeProvider";
 import { DamTable } from "./DamTable";
-function DamPage(): React.ReactElement {
-    return <DamTable />;
+
+type Props = {
+    renderContentScopeIndicator?: (scope: ContentScopeInterface) => React.ReactNode;
+};
+
+const defaultRenderContentScopeIndicator = () => (
+    <ContentScopeIndicator variant="toolbar" global>
+        <ScopeIndicatorContent>
+            <Domain fontSize="small" />
+            <ScopeIndicatorLabelBold variant="body2">
+                <FormattedMessage {...messages.globalContentScope} />
+            </ScopeIndicatorLabelBold>
+        </ScopeIndicatorContent>
+    </ContentScopeIndicator>
+);
+
+const ScopeIndicatorLabelBold = styled(Typography)`
+    && {
+        font-weight: 400;
+        padding: 0 8px 0 4px;
+        text-transform: uppercase;
+    }
+`;
+
+const ScopeIndicatorContent = styled("div")`
+    display: flex;
+    align-items: center;
+`;
+
+function DamPage({ renderContentScopeIndicator = defaultRenderContentScopeIndicator }: Props): React.ReactElement {
+    const { scope, match } = useContentScope();
+    const routeMatch = useRouteMatch();
+    const damRouteLocation = routeMatch.url.replace(match.url, "");
+    useContentScopeConfig({ redirectPathAfterChange: damRouteLocation });
```

Comment:
Here I calculate the DAM's path by "subtracting" the scope URL part (/main/en/assets - /main/en = /assets). This removes the need to specify the path twice (once for the route, once for the redirectPathAfterChange). What do you think?

Reply:
good idea if that works always